preprocessors/togeojson-gpx.preprocessor.js

```diff
@@ -102,6 +102,14 @@ module.exports = function(infile, outdirectory, callback) {
       });
     });
 
+    var archivedOriginal = path.join(outdirectory, '/archived.gpx');
+    digest(infile, function(err, callback) {
```

Comment:
No need to run `digest` again here, since the metadata file is [already added](https://github.com/mapbox/preprocessorcerer/blob/45d9f4a91153625af3748cb0c6f9a13f480453bc/preprocessors/togeojson-gpx.preprocessor.js#L96-L103).